_docs/templates/_layer_data_guide.md.jinja

```diff
@@ -0,0 +1,120 @@
+(layer-data-tuples)=
+## The LayerData tuple
```

Comment:
starting this at h2 gives me a warning (from myst) about non-consecutive header level increases.

Is there a way to silence that warning?

I'd prefer to be able to use non-consecutive header levels so I don't mind the warning itself but they poop all over the build output.

Reply:
yeah, as mentioned elsewhere,  I think jb pooping warnings all over the place is sort of par for the course.  A year or so ago I tried to clear all of our warnings out of the docs build, and now it's a superfund site again.  So I'm totally with you on this... I don't know how to silence those warnings, but I've given up on that a bit.  should be a new issue

Reply:
(this one also relates to this being an `include` ... it can't start at h1)